pipelines/HSC/ApPipe.yaml

```diff
@@ -7,18 +7,7 @@ imports:
   - location: $AP_PIPE_DIR/pipelines/HyperSuprimeCam/ApPipe.yaml
 parameters:
   # Use dataset's specific templates
-  coaddName: deep
-  # TODO: redundant connection definitions workaround for DM-30210
-  template: deepCoadd
-  diaSrcCat: deepDiff_diaSrc
-  diaSrcSchema: deepDiff_diaSrc_schema
-  diaSrcParquet: deepDiff_diaSrcTable
-  diff: deepDiff_differenceExp
-  diffScore: deepDiff_scoreExp
-  diffWarp: deepDiff_warpedExp
-  diffMatch: deepDiff_matchedExp
-  assocSrc: deepDiff_assocDiaSrc
-  # TODO: end DM-30210 workaround
+  coaddName: goodSeeing
 tasks:
   isr:
     class: lsst.ip.isr.IsrTask
```

Comment:
Having looked a bit closer, I'm pretty sure the `calibrate` override in this file is not appropriate for the new refcats -- the included file looks for catalogs called `gaia` and `panstarrs`, which is the `ap_verify` convention. I'm not sure about the `isr` override -- do we have all the brighter-fatter and transmission curve HSC calibrations now?

Reply:
You are right. I confirm that the new repo will have all those calibration data and special config overrides are no longer needed. 

I'm leaving the DECam part of the pipeline configs untouched. Despite that we don't use DECam data for testing at the moment, we might in the future (?).

Reply:
We use the DECam pipeline for unit testing; the repository for it is in `tests/data/central_repo`.